atsintegration/examples/classes/com/arjuna/jta/distributed/example/server/impl/ServerImpl.java

```diff
@@ -151,6 +151,7 @@ public void initialise(LookupProvider lookupProvider, String nodeName, int portO
 		List<String> xaResourceOrphanFilterClassNames = new ArrayList<String>();
 
 		xaResourceOrphanFilterClassNames.add("com.arjuna.ats.internal.jta.recovery.arjunacore.JTATransactionLogXAResourceOrphanFilter");
+		xaResourceOrphanFilterClassNames.add("com.arjuna.ats.internal.jta.recovery.arjunacore.JTAActionStatusServiceXAResourceOrphanFilter");
 		xaResourceOrphanFilterClassNames.add("com.arjuna.ats.internal.jta.recovery.arjunacore.JTANodeNameXAResourceOrphanFilter");
 		xaResourceOrphanFilterClassNames.add("com.arjuna.ats.internal.jta.recovery.arjunacore.SubordinateJTAXAResourceOrphanFilter");
```

Comment:
for the sake of the completeness, can you please add down here the `JTAActionStatus...` filter for subordinate transactions too? It's class `com.arjuna.ats.internal.jta.recovery.arjunacore.SubordinationManagerXAResourceOrphanFilter`